spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'user/index', type: :system do
+  before(:each) do
+    driven_by(:rack_test)
+    @user = User.first
+  end
+
+  it 'has username' do
```

Comment:
- [ ] Kindy check on your app, these four tests are failing to pass below is a screenshot of the error am getting:
![image](https://user-images.githubusercontent.com/61408860/187752549-0598dcad-a1dc-44dd-93ed-37193b06c3d7.png)